src/utils/fluent.js

```diff
@@ -57,7 +57,7 @@ function updateLocale (requestedLocales) {
 * @param {string} id - The Fluent message id.
 */
 function getRawMessage (id) {
-  const locale = localStorage.getStore().get('locale')
+  const locale = localStorage?.getStore()?.get('locale') || 'en'
```

Comment:
Good catch – this should fallback locale when not coming from a server request, which is what Jest is presumably doing.

I think just a single `?` would suffice?  `localStorage` will always be imported in this module, but the store hasn't been initialized via `run()` in `app.js` – that's only done in a request phase.

I tested, but can you confirm this works?
`localStorage.getStore()?.get('locale') || 'en'`

Reply:
Makes sense! Works.